spec/system/page_objects/components/composer.rb

```diff
@@ -219,13 +219,15 @@ def move_cursor_after(text)
           const index = composer.value.indexOf(text);
           const position = index + text.length;
 
+          composer.focus();
```

Comment:
Had to add this line otherwise doing `composer.type_content` after a `composer.move_cursor_after` was appending at the end of the `textarea` instead of at the caret position.